leda/components/Pagination/types.ts

```diff
@@ -28,6 +40,8 @@ export interface PaginationProps {
   itemsRangeInfoRender?: CustomRender<PaginationProps, PaginationState, PaginationRangeInfoProps>,
   /** Total items info customizator */
   itemsTotalInfoRender?: CustomRender<PaginationProps, PaginationState, PaginationTotalInfoProps>,
+  /** Customize component text labels */
+  messages?: PaginationMessages,
```

Comment:
What if you only want to customize 1 field? With this you'd have to provide all existing messages

Reply:
Actually yes
It's a mistake 
Will update